src/Compilers/CSharp/Portable/BoundTree/Formatting.cs

```diff
@@ -40,10 +40,7 @@ public override object Display
 
     internal sealed partial class BoundSuppressNullableWarningExpression
     {
-        public override object Display
-        {
-            get { return (object)Expression.Type ?? (Expression.Display + "!"); }
-        }
+        public override object Display => Expression.Display;
```

Comment:
>public override object Display => Expression.Display; [](start = 8, length = 53)

Aren't we losing the ```!```? Consider creating FormattableString, see the implementation for BoundStackAllocArrayCreation for example.

Reply:
The error message should be `"cannot convert from 'method group'"` rather than `"from 'method group!'"`.

---
In reply to: [187138810](https://github.com/dotnet/roslyn/pull/26724#discussion_r187138810) [](ancestors = 187138810)